Tests/PhpUnit/AbstractPrependExtensionTestCaseTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Matthias\DependencyInjectionTests\Test\DependencyInjection;
+
+use Matthias\SymfonyDependencyInjectionTest\PhpUnit\AbstractExtensionTestCase;
+use Matthias\SymfonyDependencyInjectionTest\Tests\Fixtures\MatthiasDependencyInjectionTestExtension;
+use PHPUnit\Framework\ExpectationFailedException;
+
+class AbstractPrependExtensionTestCaseTest extends AbstractExtensionTestCase
+{
+    protected function getContainerExtensions()
```

Comment:
Yes, I know, but please see: SymfonyDependencyInjectionTest/Tests/PhpUnit/AbstractExtensionTestCaseTest.php

I always follow current practice of the library's author - regardless of my own practice and coding standards. Code should look like as if it has been written as one single person, right? :)